index.js

```diff
@@ -38,12 +41,15 @@ function Scroller(scroller, content, render, isPrepend, isSticky, cb) {
       var m = queue.shift()
       var r = render(m)
       append(scroller, content, r, isPrepend, isSticky)
+      obv.set(queue.length)
     }
   }
 
   function scroll (ev) {
-    if(isEnd(scroller, buffer, isPrepend) || !isFilled(content))
+    if(isEnd(scroller, buffer, isPrepend) || !isFilled(content)) {
       pause.resume()
+      add()
```

Comment:
yeah, I think the logic around this stuff isn't quite right (who would have thought a scroller was this difficult?) and problem I have been seeing is items get added (to bottom) when you move the scroll bar, even if you are not near the bottom. this significantly impacts the frame rate (you'll see it go janky)